docker/mint/Dockerfile

```diff
@@ -39,53 +39,18 @@ RUN mkdir -p /opt/libra/bin  /libra/client/data/wallet/
 
 COPY --from=builder /libra/target/release/cli /opt/libra/bin
 COPY --from=builder /libra/target/release/libra-faucet /opt/libra/bin
-COPY --from=builder /libra/target/release/config-builder /opt/libra/bin
 COPY docker/mint/server.py /opt/libra/bin
-COPY docker/mint/docker-run.sh /opt/libra/bin
 
 # Test the docker container before shipping.
 FROM pre-test AS test
 
 #install needed tools
 RUN apt-get update && apt-get install -y procps
 
-# TEST docker-run.sh
-#   GIVEN necessary environment arguments to run docker-run.sh
-ARG AC_PORT="8080"
-ARG AC_HOST="172.18.0.13"
-ARG CFG_CHAIN_ID="TESTING"
-ARG LOG_LEVEL="info"
-ARG CFG_NUM_VALIDATORS=5
-
-#   WHEN we execute docker-run.sh
-#   THEN a gunicorn process is started
-#
-# NOTES:
-#   Attempt to find the gunicorn processes (it forks itself) with pgrep.
-#   Notice that the regex used by pgrep matches the start of a process name.
-#   We wait up to 5 seconds for the process to start, else fail.
-#   This is an imperfect test, and if the docker-run.sh exits before a process is found
-#   it may lead to race conditions.
-#
-# If the test fails we break the docker image build.
-RUN set -x; \
-    /opt/libra/bin/docker-run.sh &  \
-    PIDS=$( pgrep -f '^/usr/bin/python3 /usr/local/bin/gunicorn' | head -n 1 ); \
-    COUNTER=5; \
-    while [ -z $PIDS ] && [ $COUNTER -gt 0 ]; do \
-    sleep 1; \
-    PIDS=$( pgrep -f '^/usr/bin/python3 /usr/local/bin/gunicorn' | head -n 1 ); \
-    COUNTER=$(( $COUNTER - 1 )); \
-    done; \
-    if [ -z $PIDS ]; then exit 1; else kill -9 $PIDS || true; fi
-
 FROM pre-test as prod
 # Mint proxy listening address
 EXPOSE 8000
 
-# Define CFG_SEED, AC_HOST, AC_PORT and CFG_CHAIN_ID environment variables when running
-CMD /opt/libra/bin/docker-run.sh
```

Comment:
We can remove the config-builder but not this startup script yet. I think the faucet server is using the CLI based implementation.  cc @mgorven

Reply:
k8s doesn't use any of these wrapper scripts, this is fine to remove.

Reply:
no one is using these :/
they probably have some value in indicating what is going on, but i would propose we just update the Dockerfile to have an explicit command at some point if that's the route we want to support.... alternatively we just leverage docker-compose and slowly get that into CI. That way we ensure that our ops infra has better test coverage.

Reply:
docker compose sounds like a good follow-up.